### PR TITLE
Fixes #195 | Don't route errors in the success handler through the failure handler

### LINKED ISSUE #195: Don't route errors in the success handler through the failure handler
Currently, whenever an error happens in the success handler of the `require` call, it is routed to the failure handler and that causes confusion because it looks like something is wrong in the loader or the required modules.

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

File: src/loader/loader.js [hunk 1]
```diff
@@ -334,6 +344,28 @@ export default class Loader {
...
+	 */
+	_reportSuccessHandlerError(err) {
+		const config = this._config;
+
+		if (!config.explainResolutions) {
+			return;
+		}
+
+		console.log('---------------------------------------');
+		console.log('Liferay AMD Loader: Uhandled error in success handler');
```
Comment (wincent) [inline src/loader/loader.js]:
```suggestion
		console.log('Liferay AMD Loader: Unhandled error in success handler');
```

File: src/loader/loader.js [hunk 2]
```diff
@@ -274,7 +278,13 @@ export default class Loader {
 
 						this._setModuleImplementations(undefinedModuleNames);
 
-						success(...this._getModuleImplementations(moduleNames));
+						try {
+							success(
+								...this._getModuleImplementations(moduleNames)
```
Comment (wincent) [inline src/loader/loader.js]: I don't think there is anything in `_getModuleImplementations()` that could throw right now, but it still might be worth future-proofing this by moving the call outside the `try`.

Comment (izaera) [reply src/loader/loader.js]: Added feedback to the PR.

File: src/loader/loader.js (hunk 1 above)
Comment (izaera) [reply src/loader/loader.js]: Removed the log messages and we'll just print the plain error to the console to emulate standard browser's behaviour.

**Pull request merged**